lnst/Recipes/ENRT/GreTunnelOverVlanRecipe.py

```diff
@@ -63,6 +63,8 @@ class GreTunnelOverVlanRecipe(
     host2.eth0 = DeviceReq(label="net1", driver=RecipeParam("driver"))
     host2.eth1 = DeviceReq(label="net1", driver=RecipeParam("driver"))
 
+    vlan_id = Param(default=10)
```

Comment:
I'd prefer using the `IntParam` class to do a little bit of type checking for the parameter. It would avoid potentially situations when someone would assign a string value to the parameter.

Reply:
Done